CHANGELOG.rst

```diff
@@ -6,7 +6,9 @@ This document describes changes between each past release.
 3.1.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add a ``validate`` command that would let you validate exported
+  records of a file with the exported collection schema to validate
+  that records can be loaded. (#61)
```

Comment:
This sentence is not entirely clear. `Add a validate command to run JSON schema validation on the records locally`  or something like that?